lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -254,13 +256,7 @@ def main():
                 params['team'] = team['id']
 
             if module.params.get('ssh_key_data'):
-                filename = module.params.get('ssh_key_data')
-                if not os.path.exists(filename):
```

Comment:
Rather than remove this entirely, I would leave this in as a fallback and warn (do not fail) that this behavior will be changed in the future.